apps/docs/content/components/BreakoutBanner/index.mdx

```diff
@@ -0,0 +1,161 @@
+import dontStack from './images/dont-stack.png'
+import doLength from './images/do-length.png'
+import dontLength from './images/dont-length.png'
+import doBG from './images/do-bg.png'
+import dontBG from './images/dont-bg.png'
+import doLeadingVisual from './images/do-leading-visual.png'
+import dontLeadingVisual from './images/dont-leading-visual.png'
+import doLeadingVisualColor from './images/do-leading-visual-color.png'
+import dontLeadingVisualColor from './images/dont-leading-visual-color.png'
+
+Use breakout banners to promote important information and actions in-between blocks of content.
+
+Whereas the CTA banner is intended to redirect users to their respective conversion outlets, links in the breakout banner should only cross-link to supporting content.
+
+## Anatomy
+
+<img
+  src={anatomy}
+  alt="Breakout banner with location-based annotations for heading, description, leading visual and link groups"
+/>
+
+## Usage
+
+Breakout banners should be used in the middle of a page to break up long blocks of content. Avoid using the breakout banner at the beginning or towards the end of a page.
+
+<DoDontContainer>
+  <Do>
+    <img src={doPosition} alt="" />
+    <Caption>Use breakout banners to break up longer pages.</Caption>
+  </Do>
+  <Dont>
+    <img src={dontPosition} alt="" />
+    <Caption>
+      Don't use the breakout banner too early or late in the page (E.g. near the
+      footer). Prefer CTA banner in case of the latter. Also avoid placing the
+      breakout banner too close to related components like the CTA banner.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Alignment
+
+The breakout banner content can be aligned to the start or center of the parent element, with the former being the default. Use the center alignment if context calls for it.
+
+E.g. Use center alignment if the breakout banner is placed between two sections that are also centered aligned to avoid breaking the visual flow.
+
+<DoDontContainer>
+  <Do>
+    <img src={doAlign} alt="" />
+    <Caption>Use start alignment when possible.</Caption>
+  </Do>
+  <Dont>
+    <img src={dontAlign} alt="" />
+    <Caption>
+      Don't use start alignment when adjacent content is center aligned.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Use sparingly
+
+Breakout banners should be used sparingly, at most once per page. This is to avoid diluting the impact of the spotlighted content or links in the context of the wider page.
+
+<DoDontContainer>
+  <Do>
+    <img alt="" src={doUseSparingly} />
+    <Caption>Use breakout banner sparingly.</Caption>
+  </Do>
+  <Dont>
+    <img alt="" src={dontStack} />
+    <Caption>
+      Don't stack multiple breakout banners, prefer Bento instead.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Use concise headings
+
+Both heading and optional descriptions in the breakout banner should be short and focussed. Heading should be not be longer than 2 lines, and description should be no longer than 3-4 lines.
```

Comment:
```suggestion
Both heading and optional descriptions in the breakout banner should be short and focussed. Heading should be not be longer than 3 lines, and description should be no longer than 3-4 lines.
```

We should increase this limit to 3 lines. Even short text with longer words often requires 3 lines, as seen in the "Do" example below.

Reply:
Cool, thanks for spotting that 😅